### PR TITLE
Issue/update reader analytics

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Update analytics in the ***Reader*** tab.  One [event](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/update-reader-analytics?expand=1#diff-bd4eaa15deb1338da7ce24981478314dL842) was incorrectly tracking likes when the post was unliked.  

### Test
0. Set breakpoints on events.
1. Go to ***Reader*** tab.
2. Tap ***View*** button on post card.
3. Notice ***READER_ARTICLE_VIEWED*** is hit.
4. Tap ***Like*** button on post card.
5. Notice ***READER_ARTICLE_LIKED*** is hit.
6. Tap ***Like*** button on same post card.
7. Notice ***READER_ARTICLE_UNLIKED*** is hit.
8. Tap post content to go to detail view.
9. Tap ***Like*** button.
10. Notice ***READER_ARTICLE_LIKED_DETAIL*** is hit.
11. Tap ***Like*** button.
12. Notice ***READER_ARTICLE_UNLIKED_DETAIL*** is hit.

### CODE REVIEW

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java [hunk 1]
```diff
@@ -389,6 +393,8 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "reader_blog_previewed";
             case READER_BLOG_UNFOLLOWED:
                 return "reader_site_unfollowed";
+            case READER_ARTICLE_VIEWED:
+                return "reader_article_viewed";
```
Comment (khaykov) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: I wonder can we rename this to something different, to better distinguish it from `reader_article_opened` ? Personally, I will probably have a hard time telling the difference from analytics dashboard. WDYT?

Comment (theck13) [reply libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: I chose `READER_ARTICLE_VIEWED` to follow the naming convention of the other buttons (e.g. `READER_ARTICLE_LIKED`).  We could change it to `READER_ARTICLE_VIEW_TAPPED`, but then we should probably change all the buttons to `READER_ARTICLE_<BUTTON>_TAPPED` for consistency and that might make legacy analytics hard to track.  What do you think?

Comment (theck13) [reply libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: I chose `READER_ARTICLE_VIEWED` to follow the naming convention of the other buttons (e.g. `READER_ARTICLE_LIKED`).  We could change it to `READER_ARTICLE_VIEW_TAPPED`, but then we should probably change all the buttons to `READER_ARTICLE_<BUTTON>_TAPPED` for consistency and that might make legacy analytics hard to track.  What do you think?

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java [hunk 2]
```diff
@@ -375,10 +375,14 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
...
             case READER_ARTICLE_LIKED:
                 return "reader_article_liked";
+            case READER_ARTICLE_LIKED_DETAIL:
+                return "reader_article_liked_detail";
             case READER_ARTICLE_OPENED:
                 return "reader_article_opened";
             case READER_ARTICLE_UNLIKED:
                 return "reader_article_unliked";
+            case READER_ARTICLE_UNLIKED_DETAIL:
+                return "reader_article_unliked_detail";
```
Comment (aerych) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: To be consistent with Tracks naming conventions the "action" part of the analytic should come after the source an context.  So in this example we'd want it to be `reader_article_detail_unliked`.  Same change for the one above 🙏

Comment (theck13) [reply libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: I updated the naming in 043f0f1.

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java (hunk 1 above)
Comment (khaykov) [reply libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: How about READER_ARTICLE_VISITED or READER_ORIGINAL_ARTICLE_VISITED (VIEWED)? `Visited` corresponds to the actual button label (Visit), and it's easier to connect to in-app action when browsing tracks. I don't have strong opition about it, and we can allways add description to Tracks event :)

**Pull request merged**